stubs/html5lib/html5lib/_inputstream.pyi

```diff
@@ -72,7 +72,7 @@ class HTMLBinaryInputStream(HTMLUnicodeInputStream):
     def detectEncodingMeta(self): ...
 
 class EncodingBytes(bytes):
-    def __new__(cls, value): ...
+    def __new__(self, value): ...
```

Comment:
This change isn't strictly related to the mypy bump, but it was a very easy allowlist entry to fix, and it means that we can delete the whole allowlist for `html5lib` since the other allowlist entry is now unused as of mypy 0.981.